pkg/agent/openflow/framework.go

```diff
@@ -15,6 +15,7 @@
 package openflow
 
 import (
+	agentconfig "antrea.io/antrea/pkg/agent/config"
```

Comment:
nit: until we need to import another config package, this alias is not necessary

Reply:
updated.